### DISCUSSION TITLE
Loggin event type CLIENT_LOGIN

### DISCUSSION THREAD

**Discussion by savchenkodn79** [keycloak/keycloak#21332] [Archive]
Hello!!! Please tell me, we are using the latest version of Keycloak quarkus 21.1.1, and the KC_LOG_LEVEL parameter is INFO,org.keycloak.events:warn. How can I see the CLIENT_LOGIN event type without enabling TRACE or DEBUG. For Keycloak Wildfly we used:
/subsystem=keycloak-server/spi=eventsListener/provider=jboss-logging:write-attribute(name=properties.success-level,value=info)
/subsystem=keycloak-server/spi=eventsListener/provider=jboss-logging:write-attribute(name=properties.error-level,value=warn)

**Comment by sschu** [comment]
You can put spi-events-listener-jboss-logging-success-level=INFO into keycloak.conf or pass SPI_EVENTS_LISTENER_JBOSS_LOGGING_SUCCESS_LEVEL=INFO as environment variable.

**Reply by savchenkodn79** [reply]
Thanks! It's worked for me!

**Reply by RichieB2B** [reply]
Why is KC_SPI_EVENTS_LISTENER_JBOSS_LOGGING_SUCCESS_LEVEL=INFO needed? According to the [docs](https://www.keycloak.org/server/all-provider-config) `spi-events-listener-jboss-logging-success-level` defaults to DEBUG so when org.keycloak.events creates an INFO log it should be processed by default by the JBoss events listerer.

Like @savchenkodn79 I do not see any LOGIN events from org.keycloak.events until I set environment variable KC_SPI_EVENTS_LISTENER_JBOSS_LOGGING_SUCCESS_LEVEL=INFO

**Reply by RichieB2B** [reply]
I get it now. `spi-events-listener-jboss-logging-success-level` sets the level that will be used to create the `org.keycloak.events` log messages. The default is DEBUG so you will not see them unless you set the log level at DEBUG (or TRACE). By setting KC_SPI_EVENTS_LISTENER_JBOSS_LOGGING_SUCCESS_LEVEL=INFO the messages are logged at the INFO level so you see them with a log level of INFO.

This is explained at https://www.keycloak.org/docs/26.1.1/server_admin/#configuring-event-logging-for-a-realm and https://www.keycloak.org/docs/26.1.1/server_admin/#event-listener
